rollup.config.js

```diff
@@ -232,3 +232,6 @@ export function makeMinificationVariants(existingConfigs) {
 
   return newConfigs;
 }
+
+export const BUILD_DIR = 'build';
+export const BUNDLES_DIR = 'bundles';
```

Comment:
Stylistic nit - feel free to take or leave:

Given how often we're going to change these values, I'm starting to think making these centralized constants which we then have to import and use template strings for is kind of overkill (especially since they're both just the value with "_DIR" stuck to the end; at that point, why not do the same for the esm and cjs directories? [EDIT: Oh, actually, you did in your bash script.]). Even in this PR, there are plenty of places where these values are just written out. For the sake of simplicity, I'd just do that everywhere [EDIT: including in the bash script]. Up to you, though.

Reply:
Sure that's a fair point. For `rollup.config.js`, I guess #4710 solves this anyway so once it is merged, I can simply get rid of these two constants. As for my bash script, I think I'm gonna stick with the `BUILD_DIR` constant, as I use it in quite a few places but for esm, cjs and types, it doesn't make too much sense to keep them.